css/wsuwp-plugin-idonate-src.less

```diff
@@ -481,18 +507,15 @@
 	}
     
     @media (max-width: 420px) {
+        .gift-planning { 
+            font-size:65%;
+        }
 		button.amount-selection {
 			width: calc(33% - 5px);
 		}
 	}
 	@media screen and (-webkit-min-device-pixel-ratio: 0) {
-		select, textarea, input {
```

Comment:
In my testing on the iOS simulator, the smallest size that prevented zooming was about 15px.